lib/page/house.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Page
+  class House
+    attr_reader :country, :house
+
+    def initialize(country:, house:)
+      @country = country
+      @house   = house
+    end
```

Comment:
you shouldn't need to instantiate with both of these. Passing just the `house` would be cleaner, and the `country` can trivially be derived from that.